src/main/resources/logsForLocalDev.json

```diff
@@ -1871,5 +1890,28 @@
       "line": "59",
       "function": "execute"
     }
+  },
+  {
+    "insertId": "123454d20006fcaeaf8dcc52",
+    "details": {
+      "message": "Instance startup: 954845e59212724cae605b074c8c7f5c",
+      "event": "INSTANCE_LOG",
+      "instanceId": "954845e59212724cae605b074c8c7f5c08f7d6cffad8638bf19bbf88d7dab5a1da1921b1",
+      "instanceEvent": "STARTUP"
+    },
+    "resourceIdentifier": {
+      "module_id": "default",
+      "project_id": "teammates-john",
+      "zone": "asia-east2-3",
+      "version_id": "8-0-0"
+    },
+    "severity": "INFO",
+    "logName": "projects/teammates-john/logs/stdout",
+    "trace": "951f19331e63e19deebf52e6c21577e5",
+    "sourceLocation": {
+      "file": "teammates.main.Application",
+      "line": "59",
+      "function": "execute"
+    }
```

Comment:
Since the start up and shut down logs are logged with `System.out.println`, all the details might become the log message of the log instead of having this kind of structure. As I do not have staging anymore, I cannot verify this. But it does not seem to affect anything for now so I think it is fine.

Reply:
Structured logging takes in any JSON-formatted message, regardless of source. In fact you might be surprised to know that the logger is configured to dump JSON-formatted messages exactly like this.

<img width="634" alt="Screenshot 2021-08-17 at 11 45 39 PM" src="https://user-images.githubusercontent.com/7261051/129758414-9377718e-8c67-42ac-bfbf-54a8b8b8795e.png">

It actually does mean wherever we use logger we can actually use `System.out.println`, but using logger is both more professional and more configurable.